### DISCUSSION TITLE
天翼云添加后报错

### DISCUSSION THREAD

**Discussion by tbc0309** [AlistGo/alist#477] [Q&A] [answered]
### Alist Version / Alist 版本

v2.0.9

### Describe the bug / 问题描述

状态：Post "https://api.xhofe.top/ocr/file/json": x509: certificate has expired or is not yet valid: current time 2022-02-04T17:51:10+08:00 is after 2021-09-30T14:01:15Z

**Comment by simpleandstupid** [comment, marked as answer]
猜测是环境太旧，受到dst root x3过期的影响，
https://letsencrypt.org/zh-cn/docs/dst-root-ca-x3-expiration-september-2021/
你需要更新ca-certificates，或者升级openssl
https://www.openssl.org/blog/blog/2021/09/13/LetsEncryptRootCertExpire/

**Reply by xhofe** [reply]
目测应该是因为这个
